.github/workflows/runtime-tests.yml

```
@@ -72,6 +72,9 @@ jobs:
         run: |
           cd ./Source/DafnyRuntime/DafnyRuntimeJs
           make all
+      - name: Install Miri for DafnyRuntimeRust
+        run: |
+          rustup +nightly-x86_64-pc-windows-gnu component add miri
```

The Miri docs recommend this to keep test output clean: https://github.com/rust-lang/miri?tab=readme-ov-file#running-miri-on-ci

```suggestion
          rustup +nightly-x86_64-pc-windows-gnu component add miri
          cargo miri setup
```